.travis.yml

```diff
@@ -8,10 +8,7 @@ cache:
 
 before_install:
   - phpenv config-rm xdebug.ini || true
-  - |
-    if [ "x$COVERAGE" == "xyes" ]; then
-      pecl install pcov-1.0.0
-    fi
+  - pecl install pcov-1.0.0
```

Comment:
Can we do just `pecl install pcov` and use the latest stable? It may be required for compatibility with the latest PHP versions.